client/app/main.py

```diff
@@ -49,10 +53,9 @@
                 output_dict[event.code] = event.state
 
                 if output_dict["BTN_EAST"] == 1:
```

Comment:
could you please remind me what button is this one? 😂

Reply:
The red one https://confluence.zeppelin.com/display/ZEPTDA/Gamepad

Reply:
thanks!